src/output/tecplotIO.F90

```diff
@@ -59,7 +62,9 @@ module tecplotIO
 
   ! Tecplot Variable names of the data in the lift distribution data file:
   character(len=maxCGNSNameLen), dimension(:), allocatable :: liftDistName
-  integer(kind=intType), parameter :: nLiftDistVar=18
+  !integer(kind=intType), parameter :: nLiftDistVar=18
```

Comment:
you can remove the comments here, and just keep the declaration of nliftdistvar=21. we added 3 to this because we added 3 additional outputs